solitude/settings/base.py

```diff
@@ -355,9 +355,18 @@
     '--logging-filter=xsd',
     '--with-nicedots',
     '--with-blockage',
-    '--http-whitelist=""',
 ]
 
+if not os.environ.get('LIVE_TESTS'):
```

Comment:
We could test specific values, but testing for the presence of the env is enough and pretty common.